aspnetcore/mvc/controllers/filters.md

```diff
@@ -14,68 +14,64 @@ uid: mvc/controllers/filters
 ---
 # Filters
 
-By [Steve Smith](http://ardalis.com)
+By [Tom Dykstra](https://github.com/tdykstra/) and [Steve Smith](http://ardalis.com)
 
-*Filters* in ASP.NET MVC allow you to run code before or after a particular stage in the execution pipeline. Filters can be configured globally, per-controller, or per-action.
+*Filters* in ASP.NET Core MVC allow you to run code before or after certain stages in the request processing pipeline. Built-in filters handle tasks such as authorization (preventing access to resources a user isn't authorized for), ensuring that all requests use HTTPS, and response caching (short-circuiting the request pipeline to return a cached response). You can create custom filters for work specific to your application, such as error handling.
 
```

Comment:
We should mention `cross-cutting-concerns` here instead of just saying 'work'. Filters are **the** solution for anything that you don't want to duplicate across actions.